src/utils/version.js

```diff
@@ -64,6 +64,24 @@ const version = {
     } else {
       return false;
     }
+  },
+
+  /**
+   * Checks if custom tokens are allowed.
```

Comment:
on ledger

Reply:
Done in [86d6ee0](https://github.com/HathorNetwork/hathor-wallet/pull/264/commits/86d6ee02563d136387db484d7848ea2da6a5a3a4)